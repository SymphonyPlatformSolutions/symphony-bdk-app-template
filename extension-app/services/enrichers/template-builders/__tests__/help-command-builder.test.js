import HelpCommandBuilder from '../help-command-builder';

const helpCommandData = {
  commands: '<br /> <b>Example:</b> @Example &lt;comand example&gt;',
};

describe('Help Command Builder', () => {
  it('Should build message', () => {
    expect(HelpCommandBuilder.build(helpCommandData)).toMatchSnapshot();
  });
});
