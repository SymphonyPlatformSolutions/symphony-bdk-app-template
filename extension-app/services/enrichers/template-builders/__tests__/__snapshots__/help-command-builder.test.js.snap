// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Help Command Builder Should build message 1`] = `
"<messageML>
  <card class=\\"barStyle\\" data-icon-src=\\"\\">
    <header>
      <img style=\\"width: 12px; height: 12px; position: relative; top: 1px;\\" src=\\"help-command-icon.svg\\" />
      See below the commands you can use to interact with the Template Bot.
    </header>

    <body>
      <div>
        <div style=\\"padding-bottom: 4px;\\"><b style=\\"color: \\">Commands</b></div>
        <div style=\\"border-left: 2px solid ;\\">
          <div style=\\"padding-left: 4px;\\">
            <b>Example:</b> @Example &lt;comand example&gt; <br />
          </div>
        </div>
      </div>
      <div style=\\"margin-top: 8px\\">
      </div>
    </body>
  </card>
</messageML>"
`;
